ui/qt-lib/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs

```diff
@@ -223,6 +225,11 @@ handleWalletInfoEvent UiLangFace{..} ev = do
         DoDelete -> putMVar resultVar True
         Cancel -> putMVar resultVar False
 
+    WalletInfoConfirmSend resultVar sendInfo ->
+      liftIO $ runConfirmSend sendInfo >>= \case
+        ConfirmationAccepted -> putMVar resultVar True
```

Comment:
In the future this dialog will have to send 'confirm' event some way, then wait for 'tx sent' event from backend, and run the spinner until then. Just to remind.